src/welle-gui/QML/settingpages/RTLSDRSettings.qml

```diff
@@ -18,7 +18,7 @@ SettingSection {
     WSwitch {
         id: enableBiasTee
         Layout.fillWidth: true
-        text: qsTr("Enable bias tee (not from all dongles supported)")
+        text: qsTr("Enable bias tee (not supported by all dongles)")
```

Comment:
End sentence in "."

Reply:
good point, I guess not. But seeing the position of said button, the text seems to be too long (not your fault).